src/helpers/URLHelper.js

```diff
@@ -0,0 +1,10 @@
+var uri = require("URIjs");
+
+export function updateApplicationUrlState(url) {
```

Comment:
Also helps for when I allow multiple tables without url persistence, I can store and update the url in localStorage.